zap/src/lib.rs

```diff
@@ -1,7 +1,7 @@
-mod config;
-mod irgen;
-mod output;
-mod parser;
+pub mod config;
+pub mod irgen;
+pub mod output;
+pub mod parser;
```

Comment:
`irgen` tests need to access this and I think it's alright to have these public as we don't publish to cargo